bin/test

```diff
@@ -11,7 +11,7 @@
 
 ARGUMENTS=$@
 if [ -z "$ARGUMENTS" ]; then
-    ARGUMENTS=`go list ./...`
+    ARGUMENTS=`go list ./... | sed '/e2e/d'` #skip e2e package - integration tests by default
```

Comment:
Go test has some tags feature, would be good to investigate that

Reply:
Tags have some limitations when it comes for full package of tests switching of. Tried that - does not help

Reply:
Anyway would be nice to filter by full package name

Reply:
Yea will need to investigate further - maybe some other time.